api/integrations/github/views.py

```diff
@@ -46,6 +50,31 @@ def wrapper(request, organisation_pk):
     return wrapper
 
 
+def github_api_call_error_handler(
+    error: str | None = None,
+) -> Callable[..., Callable[..., Any]]:
+    def decorator(func):
+        @wraps(func)
+        def wrapper(*args, **kwargs) -> Response:
+            try:
+                return func(*args, **kwargs)
+            except requests.RequestException as e:
+                return Response(
```

Comment:
I'd throw in a `logger.error`/`logger.warning` here to log the GH error for our own convenience.

Reply:
Done